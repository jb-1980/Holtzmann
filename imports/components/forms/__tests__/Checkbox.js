import renderer from "react-test-renderer";
import Checkbox from "../Checkbox";

it("should exist", () => {
  const checkbox = renderer.create(<Checkbox />);
  expect(checkbox).toMatchSnapshot();
});

it("should accept props", () => {
  const checkbox = renderer.create(
    <Checkbox
      validation={() => {}}
      status="test2"
      disabled
      errorText="test4"
      error
      classes={["test5"]}
      id="test7"
      defaultValue="checked"
      clicked={() => {}}
      style="test9"
      inputClasses={["test10", "test11"]}/>);
  expect(checkbox).toMatchSnapshot();
});
