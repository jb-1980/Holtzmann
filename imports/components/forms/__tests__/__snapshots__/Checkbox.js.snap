exports[`checkbox should exist 1`] = `undefined`;

exports[`test should exist 1`] = `
<div
  className="input checkbox">
  <h6
    className="soft-left push-half-left flush-bottom text-left float-left locked-top">
    <small />
  </h6>
  <input
    className={undefined}
    defaultChecked=""
    disabled={undefined}
    id={undefined}
    name={undefined}
    onClick={undefined}
    style={
      Object {
        "width": 0
      }
    }
    type="checkbox" />
  <label
    htmlFor={undefined}
    style={Object {}} />
</div>
`;
